data/json/monsters.json

```diff
@@ -2473,6 +2473,37 @@
     "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "BONES"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_zombie_hollow",
+    "name": "zombie hollow",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"white",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":20,
+    "aggression":100,
+    "morale":100,
+    "speed":90,
+    "melee_skill":6,
+    "melee_dice":2,
+    "melee_dice_sides":6,
+    "melee_cut":4,
+    "dodge":4,
+    "armor_bash":8,
+    "armor_cut":8,
+    "vision_day":30,
+    "vision_night":3,
+    "item_chance":80,
+    "death_drops": "mon_zombie_soldier_death_drops",
```

Comment:
Hmm.  Are these an upgraded soldier, and if so, would the armor not serve as an exoskeleton of sorts (and deserve mention in the blurb?)